packages/dashboard-frontend/src/services/widget-factory.js

```diff
@@ -15,16 +15,19 @@ import { SearchRankingCompareWidget } from "../widgets/search-ranking-compare-wi
 export class WidgetFactory {
 	#dataProvider;
 	#remoteDataProvider;
+	#remoteCachedDataProviders;
 	#dataFormatters;
 
 	/**
 	 * @param {import("./data-provider").DataProvider} dataProvider
 	 * @param {import("./remote-data-provider").RemoteDataProvider} remoteDataProvider
+	 * @param {object<import("./remote-cached-data-provider").RemoteCachedDataProvider>} remoteCachedDataProviders
```

Comment:
```suggestion
	 * @param {Object<WidgetType,import("./remote-cached-data-provider").RemoteCachedDataProvider>} remoteCachedDataProviders A remote cached data provider per widget type.
	 */
```

(and yes, below is wrong with the lower case o)

Reply:
haha ok, let's fix in both places then. Done [here](https://github.com/Yoast/wordpress-seo/pull/22186/commits/4bea8da59ce30aa692e65ab9a519f2672f67f17b).